docker/scripts/install-pyenv.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# For managing all the local python installations for testing, use pyenv
+curl -L https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer | bash
+
+# To enable testing versions like 3.4.8 as 3.4 in tox, we need to alias
+# pyenv python versions
+git clone https://github.com/s1341/pyenv-alias.git $(pyenv root)/plugins/pyenv-alias
+
+# Python versions to test against
+VERSION_ALIAS="python3.6" pyenv install 3.6.9
+# Possible to add more versions like this: VERSION_ALIAS="python3.7" pyenv install 3.7.5
```

Comment:
Out of curiosity, would we want to test with Python 3.8? Haha it seems like the upgrade to 3.8 broke some things (as observed with MA and smtool).